packages/mdc-segmented-button/segment/_segment.scss

```diff
@@ -22,14 +22,21 @@
 
 // Core: Define complete styles to render component segment, including base and theme styles
 
-@use '@material/feature-targeting';
-@use '@material/typography/mixins' as typography-mixins;
+@use '@material/feature-targeting/feature-targeting';
+@use '@material/typography/typography';
+@use '@material/touch-target/touch-target';
+@use '@material/elevation/elevation';
+@use '@material/elevation/elevation-theme';
 @use './segment-theme';
 
-@mixin core-styles($query: feature-targeting.all()) {
+@mixin without-ripple($query: feature-targeting.all()) {
```

Comment:
Since we've moved with-ripple() styles to separate module let's rename this back to `core-styles()`

Reply:
Then in `segment/styles.scss` should I keep the mixin as `core-styles`? Or should I remove `segment/styles.scss` and just put those 2 includes in `./styles.scss`?